Kitodo/pom.xml

```diff
@@ -401,20 +449,32 @@
             <artifactId>kitodo-contentserver</artifactId>
             <version>2.0.0</version>
         </dependency>
-        <dependency>
-            <groupId>org.richfaces.framework</groupId>
-            <artifactId>richfaces-api</artifactId>
-            <version>3.1.6.GA</version>
-        </dependency>
```

Comment:
Removing this `richfaces-api` cause a regression on running the application. Reason: ClassNotFound exception for `org.ajax4jsf.application.ViewHandlerWrapper`. Better solution could be an exclusion of every dependency.